prody/proteins/compare.py

```diff
@@ -336,8 +336,10 @@ def buildFromChain(self, chain):
             aa = AAMAP.get(res.getResname(), 'X')
             if aa == '-':
                 aa = 'X'
-            if aa == 'X':
+            if aa == 'X' and res.getResname() != 'HOH':
```

Comment:
This rule looks incomprehensive... Even TIP3P waters could escape from this check.

Reply:
Good point. I wonder if OpenMM or some other package has way of representing solvent